investigator_message_metrics/overlays/prod/thoth-notification.yaml

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
@KPostOffice let remove the prod overlay it is not needed anymore.

Reply:
Oops yeah, sorry missed that one. It's gone now